schemas/comorbidity.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "comorbidity",
+  "description": "The collection of data elements related to a donor's comorbidities",
+  "meta": {
+     "parent": "donor"
+  },
+  "fields": [
+    {
+      "name": "program_id",
+      "valueType": "string",
+      "description": "Unique identifier of the ARGO program.",
+      "meta": {
+        "validationDependency": true,
+        "primaryId": true,
+        "foreignKey": "sample_registration.program_id",
+        "displayName": "Program ID"
+      },
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "submitter_donor_id",
+      "valueType": "string",
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "meta": {
+        "validationDependency": true,
+        "primaryId": true,
+        "foreignKey": "sample_registration.submitter_donor_id",
+        "displayName": "Submitter Donor ID"
+      },
+      "restrictions": {
+        "required": true,
+        "regex": "#/regex/submitter_id"
+      }
+    },
+    {
+      "name": "prior_malignancy",
+      "description": "Prior malignancy affecting donor.",
+      "restrictions": {
+        "codeList": "#/list/yes_no"
+      },
+      "valueType": "string",
+      "meta": {
+        "displayName": "Prior Malignancy"
+      }
+    },
+    {
+      "name": "laterality_of_prior_malignancy",
+      "description": "If donor has history of prior malignancy, indicate laterality of previous diagnosis. (Reference: caDSR CDE ID 4122391)",
+      "valueType": "string",
+      "restrictions": {
+        "codeList": ["Bilateral", "Left", "Midline", "Not applicable", "Right", "Unilateral, side not specified", "Unknown"],
+        "script": ""
+      },
+      "meta": {
+        "displayName": "Laterality at Prior Malignancy"
+      }
+    },
+    {
+      "name": "age_at_cormorbidity_diagnosis",
+      "valueType": "integer",
+      "description": "Indicate the age of comorbidity diagnosis, in years.",
+      "restrictions": {
+         "script": ""
+      },
+      "meta": {
+        "units": "years",
+        "displayName": "Age at Comorbidity Diagnosis"
+      }
+    },
+    {
+      "name": "comorbidity_type_code",
+      "valueType": "string",
+      "description": "Indicate the code for the cormorbidity using the WHO ICD-10 code classification (https://icd.who.int/browse10/2019/en).",
+      "restrictions": {
+        "regex": "^[A-Z][0-9]{2}(.[0-9]{1,3}[A-Z]{0,1})?$"
+      },
+      "meta": {
+        "examples": "E10, C50.1, I11, M06",
+        "displayName": "Comorbidity Type Code"
+      }
+    },
+    {
+      "name": "comorbidity_treatment",
```

Comment:
is this helpful without extended information? should there be an open text field to describe what the comorbidity treatmetn is?

Reply:
Yes ieven with "comorbidity_treatment_status" (I changed the field name) on its own, it's still useful. Knowing whether a comorbidity is controlled or not can affect the severity of cancer. For example, this study (https://pubmed.ncbi.nlm.nih.gov/18409001/) mentions the severity of colorectal cancer in uncontrolled diabetes vs. controlled diabetes. "_Siddiqui et al found that patients with uncontrolled type II diabetes (high hemoglobin 1c [HbA1c] levels before diagnosis) tended to have more advanced colorectal cancer diagnosed at a younger age than those with well controlled diabetes despite having similar sociodemographic characteristics and number of outpatient visits._" (Reference: https://acsjournals.onlinelibrary.wiley.com/doi/pdf/10.3322/caac.21342)

I added a field (free text) for data submitters to enter treatment information for comorbidities ("comorbidity_treatment"), but I did not make it required because even with just "comorbidity_treatment_status" on its own is still helpful.